ramls/customFieldOptions.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Custom Field Options Schema",
+  "additionalProperties": false,
+  "properties": {
+    "values": {
+      "type": "array",
+      "description": "Defines the values for the custom field",
+      "items": {
+        "type": "string"
+      },
+      "example": "[\"Engineering\"]"
+    },
+    "sortingOrder": {
+      "type": "string",
+      "enum": ["ASC", "DESC", "CUSTOM"],
+      "description": "Defines sorting order for the custom field",
+      "example": ""
+    },
+    "sorted": {
+      "type": "array",
+      "description": "Defines visibility of the custom field",
```

Comment:
how does this value define visibility of the custom field?

Reply:
this was a typo, will fix it